src/components/Chart/BarChart/components/CustomLabel/CustomLabel.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { LabelProps } from 'recharts';
+import useTheme from 'hooks/useTheme';
+
+interface CustomLabelProps extends LabelProps {
+  colors: Record<string, string>;
+}
+
+const CustomLabel: React.FC<CustomLabelProps> = ({ colors, value, x, y, width, height }) => {
+  const theme = useTheme();
+  const xValue = x && width ? x + width + 16 : 16;
```

Comment:
Try to give a name in "magic numbers" (ex. for number 16, 2, and 0) to improve readability  😊✌️.